### PR TITLE
Deepautoqsar edit

### PR DESCRIPTION
**Pull request by leelaprabhu**
- SDF reader
- Corrected Concat and updated format
- Corrected GraphGather
- Modified Dropout

### CODE REVIEW

File: deepchem/models/tensorgraph/layers.py [hunk 1]
```diff
@@ -507,17 +507,22 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
...
     super(Concat, self).__init__(**kwargs)
 
   def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     inputs = self._get_input_tensors(in_layers)
     if len(inputs) == 1:
       self.out_tensor = inputs[0]
       return self.out_tensor
-
-    out_tensor = tf.concat(inputs, axis=self.axis)
+
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/layers.py:520]:
```python
out_tensor = tf.concat(inputs, axis=self.axis)
if self.l2_normalize:
  out_tensor = tf.nn.l2_normalize(out_tensor, axis=self.axis)
```

File: deepchem/utils/save.py [hunk 2]
```diff
@@ -67,21 +67,27 @@ def load_data(input_files, shard_size=None, verbose=True):
       yield load_pickle_from_disk(input_file)
 
 
-def load_sdf_files(input_files, clean_mols):
+def load_sdf_files(input_files, clean_mols, only_sdf=False):
   """Load SDF file into dataframe."""
```
Comment (lilleswing) [inline deepchem/utils/save.py:71]: put the only_sdf check at the beggining of the function:
``` python
if only_sdf:
  return list()
```
